lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -542,6 +552,38 @@ def error_code_from(response)
         error_code ||= STANDARD_ERROR_CODE_MAPPING[code]
         error_code
       end
+
+      def tokenize_bank_account(bank_account, options = {})
+        account_holder_type = bank_account.account_holder_type == "personal" ? "individual" : "company"
```

Comment:
That's a great idea :+1: I'll make a constant lookup and give it the key instead of having a ternary.